src/pages/items/productId/ProductId.jsx

```diff
@@ -0,0 +1,109 @@
+import { API_USEDS_PRODUCT_INFORMATION, API_USER_COMMENTS } from "config/api";
+import styles from './ProductId.module.css';
+import Line from "components/items/productId/Line";
+import { ReactComponent as NoInquire } from 'assets/imgs/no_inquire.svg';
+
+const ProductId = () => {
+  const {
+    productName,
+    productPrice,
+    productDescription,
+    productImage,
+    productTags,
+    productFavoriteCount,
+    commentData,
+    updateAt,
+    ownerIamge,
+    ownerNickname,
+    setProductName,
+    setProductPrice,
+    setProductDescription,
+    setProductImage,
+    setProductTags,
+    setProductFavoriteCount,
+    setCommentCount,
+    setCommentData,
+  } = useProductIdState();
+
+  const location = useLocation();
+  const id = location.state.product.id;
+
+  useEffect(() => {
+    const asyncFunction = async () => {
+      const response = await fetchData(`${API_USEDS_PRODUCT_INFORMATION}/${id}`, {productId: id});
+      const product = response.data;
+      setProductName(product.name);
+      setProductPrice(product.price);
+      setProductDescription(product.description);
+      setProductImage(product.images);
+      setProductTags(product.tags);
+      setProductFavoriteCount(product.favoriteCount);
+    }
+
+    asyncFunction();
+
+  }, [id, setProductName, setProductPrice, setProductDescription, setProductImage, setProductTags, setProductFavoriteCount]);
+
+  useEffect(() => {
+    const asyncFunction = async () => {
+      const response = await fetchData(`${API_USER_COMMENTS}/${id}/comments`, {productId: id, limit: 3});
+      setCommentCount(response.data.list.length);
+      setCommentData(response.data.list);
+    }
+
+    asyncFunction();
+
+  }, [id, setCommentCount, setCommentData]);
+
+  return (
+    <div>
+      <Header />
+      <div className={styles['card-container']}>
+        <ProductImage productImage={productImage} />
+        <ProductSummary
+          productName={productName}
+          productPrice={productPrice}
+          productDescription={productDescription}
+          productTags={productTags}
+          productFavoriteCount={productFavoriteCount}
+          ownerNickname={ownerNickname}
+          ownerIamge={ownerIamge}
+          updateAt={updateAt}
+        />
+      </div>
+      <Line marginTop={40} marginBottom={40}/>
+      <Inquire />
+      {
+        commentData && commentData.map((user, index) => (
+          <>
+            <InquireItem
+              key={index}
+              content={user.content}
+              userNickname={user.writer.nickname}
+              userImage={user.writer.image}
+              updateAt={user.updatedAt}
+              commentData={commentData}
+              setCommentData={setCommentData}
+              />
+            <Line marginTop={12} marginBottom={24}/>
+          </>
+        ))
+      }
+      {
+        commentData.length <= 0 && <div className={styles['no-inquire']}><NoInquire />아직 문의가 없어요</div>
+      }
```

Comment:
P2:
해당 페이지에서 comment부분과 상품 데이터 부분을 분리하면 위의 로직들도 분리되어서 더 깔끔할 것 같습니다.
```suggestion
     <ProductComment /> // 컴포넌트 분리하며 관련 데이터 호출 함수도 분리
```

Reply:
만약 dataFetch를 page에서만 하기로 규칙을 세우신 거라면 코멘트와 상품 관련 부분을 나눠 커스텀 훅으로 분리하시면 더 깔금할 것 같습니다.